crates/bevy_pbr/src/bloom/bloom.wgsl

```diff
@@ -0,0 +1,131 @@
+#import bevy_core_pipeline::fullscreen_vertex_shader
+
+struct Uniforms {
+	threshold: f32,
+	knee: f32,
+	scale: f32,
+};
+
+@group(0) @binding(0) var org: texture_2d<f32>;
+@group(0) @binding(1) var org_sampler: sampler;
+@group(0) @binding(2) var<uniform> uniforms: Uniforms;
+@group(0) @binding(3) var up: texture_2d<f32>;
+
+fn quadratic_threshold(color: vec4<f32>, threshold: f32, curve: vec3<f32>) -> vec4<f32> {
+    let br = max(max(color.r, color.g), color.b);
+
+    var rq: f32 = clamp(br - curve.x, 0.0, curve.y);
+    rq = curve.z * rq * rq;
+
+    return color * max(rq, br - threshold) / max(br, 0.0001);
+}
+
+// samples org around the supplied uv using a filter
```

Comment:
I'm not sure what you mean by "samples org". What does org mean here?

Reply:
I think original? Naming, especially in the shader, is something I need to clean up. I didn't write this code.

Reply:
Ah, right, I forgot this was based on another PR.

Reply:
'organised' perhaps?